bluebrain/deployment/bin/configure-compilers

```diff
@@ -20,18 +20,16 @@ fi
 # Find the location of *any* GCC, disregarding the environment that's
 # active (-E).  Prefer the hash passed through by earlier stages, if
 # not available fall back to the crude version.
-LEGACY_GCC_DIR=$(spack -E location --install-dir ${LEGACY_GCC_HASH:-gcc@${LEGACY_GCC_VERSION}})
 DEFAULT_GCC_DIR=$(spack -E location --install-dir ${DEFAULT_GCC_HASH:-gcc@${DEFAULT_GCC_VERSION}})
 # Use the Spack configuration to grab the matching module for GCC;
 # requires GCC to be compiler found beforehand.
-GCC_MODULES="$(spack -E config get compilers|sed -n "
-    /^\s*spec: gcc@${LEGACY_GCC_VERSION}/,/^\s*modules:/ {
-        /^\s*modules:/ {
-            s/^\s*modules: \[\(.*\)\]\$/\1/;
-            p;
-        }
-    }")"
-log "...using gcc ${LEGACY_GCC_VERSION} from ${LEGACY_GCC_DIR} for Intel (legacy) and NVHPC with module(s) ${GCC_MODULES}"
+#GCC_MODULES="$(spack -E config get compilers|sed -n "
+#    /^\s*spec: gcc@${LEGACY_GCC_VERSION}/,/^\s*modules:/ {
+#        /^\s*modules:/ {
+#            s/^\s*modules: \[\(.*\)\]\$/\1/;
+#            p;
+#        }
+#    }")"
```

Comment:
`nitpick`: don't want you to just remove this?

Reply:
IIUC @matz-e preferred to keep this part lying around